### PR TITLE
[ET-1331] Add anonymous purchases notice

### PR DESCRIPTION
**Pull request by codingmusician**
### 🎫 Ticket

[ET-1331] 

### 🗒️ Description

When Tickets Commerce is enabled, but the setting isn't set to require users to log in before purchasing tickets, we want to show a notice, recommending they set it.

### 🎥 Artifacts 

https://imgur.com/PEUjExW.png

### ✔️ Checklist
- [x] I've included a changelog entry in the readme.txt file. 
- [x] My code is tested. 
- [x] My code has [proper inline documentation](https://the-events-calendar.github.io/products-engineering/docs/code-standards/).

[ET-1331]: https://theeventscalendar.atlassian.net/browse/ET-1331?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by codingmusician** [comment]
Note: The link URL will need to be changed. We will get that from the content team soon.

File: readme.txt [hunk 1]
```diff
@@ -179,6 +179,10 @@ Check out our extensive [knowledgebase](https://evnt.is/18wm) for articles on us
 
 == Changelog ==
 
+= [TBD] TBD =
+
+* Enhancement - Added a warniong when Tickets Commerce is enable, but users aren't required to log in before purchasing tickets. [ET-1352]
```
Comment (juanfra) [inline readme.txt:184]:
```suggestion
* Enhancement - Added a warning when Tickets Commerce is enabled, but users aren't required to log in before purchasing tickets. [ET-1352]
```

File: src/Tickets/Commerce/Admin/Notices.php [hunk 2]
```diff
@@ -34,6 +35,66 @@ public function register() {
...
+	 */
+	public function should_render_anonymous_purchases_notice() {
+		// If we're not on our own settings page, bail.
+		if ( Tribe__Settings::$parent_slug !== tribe_get_request_var( 'page' ) ) {
+			return false;
+		}
+
+		$options = get_option( Tribe__Main::OPTIONNAME, [] );
+		
+		if(
```
Comment (juanfra) [inline src/Tickets/Commerce/Admin/Notices.php:62]:
```suggestion
		if ( 
```

**Review by juanfra — COMMENTED** [review]
🎉 will get you the link today. Thanks!

**Review by redscar — APPROVED** [review]

**Pull request merged**